src/components/data-table/data-table.config.js

```diff
@@ -486,5 +486,34 @@ module.exports = {
         sort: true,
       },
     },
+    {
+      name: 'compat',
```

Comment:
This is here for my testing/so we don't regress in the future. Totally can be removed though.